content/docs/use-cases/ci-and-ml.md

```diff
@@ -0,0 +1,172 @@
+# CI and ML
+
+ML engineers and data scientists face many unique challenges. Some of these can
+be solved by adopting well-established concepts from traditional software
+engineering and development. A good example is using version control both for
+the code and for the [data](/doc/use-cases/versioning-data-and-model-files). A
+less common adoption is
+[unit testing](https://en.wikipedia.org/wiki/Software_testing#Testing_levels)
+the various stages of a data processing pipeline. Here is a non exhaustive list
+of a few other unique challenges:
+
+- Automatically re-train models upon changes
+- Systematically compare (and report) performance of different versions of a
+  model
+- Test data correctness when new datasets are introduced
+- Train a model which is computationally intensive or requires special
+  environment (e.g. GPU)
+
+[CI](https://en.wikipedia.org/wiki/Continuous_integration) is a core principal
+of nowadays software development in general and DevOps in particular. In a
+nutshell, it is aimed at frequent code, data, and model check-ins, automatic
+testing, and fast feedback. DVC provides a solid basis for automating model
+training and testing with constructs like pipelines and metrics. To streamline
+DVC into CI tools we've built a complementary tool, Continuous Machine Learning
+([CML](http://www.cml.dev)). Putting these tools together allows the bulding and
+**coding** of flows that can help tackling the aforementioned problems and many
+others. We will next dive a little deeper and outline how to harness these tools
```

Comment:
Looks good! Let's add a line break somewhere for readability- maybe put "We will next dive a little deeper...." after a break?

Reply:
This will become a paragraph with a single sentence. Is it desirable?

Reply:
Personally I am okay with it! I use single sentence paragraphs. It is a matter of personal preference.